pilot/lib/pilot/build_manager.rb

```diff
@@ -14,8 +14,9 @@ def upload(options)
       dir = Dir.mktmpdir
 
       platform = fetch_app_platform
+      ipa_path = File.expand_path(config[:ipa])
```

Comment:
Any chance you would want to move this into the `generate` method on `IpaUploadPackageBuilder` to let that method handle the expansion?